packages/ember-views/lib/mixins/view_support.js

```diff
@@ -620,7 +620,7 @@ export default Mixin.create({
     this._super(...arguments);
 
     if (!this.elementId && this.tagName !== '') {
-      this.elementId = guidFor(this);
```

Comment:
This is the wrong place to do it IMO. I don't think we want to actually allow setting `this.id`  on the instance – we have to rename the `id` arg to `elementId` [on the way in](https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/lib/components/curly-component.js#L50).